pkgs/development/haskell-modules/hackage-packages.nix

```
@@ -71436,7 +71436,7 @@ self: {
        description = "Language Server Protocol (LSP) server for Dhall";
        license = stdenv.lib.licenses.mit;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
+       broken = false;
```

```suggestion
       broken = true;
```

Generally when sending PRs for Haskell stuff, we don't directly edit the `hackage-packages.nix` file, but instead wait for the daily updater service to regenerate it.